app/Http/Controllers/Dashboard/AuthorController.php

```diff
@@ -2,21 +2,25 @@
 
 namespace App\Http\Controllers\Dashboard;
 
-use App\Models\Author;
 use App\Http\Requests\AuthorRequest;
 use App\Http\Controllers\Controller;
-use Illuminate\Support\Facades\Cache;
+use App\Repositories\Interfaces\AuthorRepositoryInterface;
```

Comment:
interface này dùng làm gì e

Reply:
em dùng bind interface to implementation ạ, ví dụ sau này có đổi cách impelement (k dùng ORM nữa) thì chỉ cần bind lại trong service provider mà k cần phải đổi ở những chỗ dùng nó ạ